docs/content/stable/explore/going-beyond-sql/tablespaces.md

```diff
@@ -223,6 +223,36 @@ yugabyte=# SELECT * FROM multi_region_table;
 Time: 337.154 ms
 ```
 
+### Using wildcards for zones
+
+In certain cases, it may be necessary to use the wildcard `*` for a zone in the placement block list. It is mostly useful when there are more than `RF` zones in the cluster, where `RF` is the desired placement of the table. In such cases, the wildcard `*` allows choosing for any of the zones in a given region, or any of the regions in a given cloud. 
+
+For example, assume a cluster is spread out over regions `us-east-1` (AZs `us-east-1a`, `us-east-1b`), `us-central-1` (AZs `us-central-1a`, `us-central-1b`), `us-west-1` (AZs `us-west-1a`, `us-west-1b`) and we wish to create an RF3 tablespace. An example tablespace like below allows copies to be placed in each of the three regions without specifying the exact AZs in those regions. This can increase the resilience of the cluster compared to explicitly specifing exactly one AZ per region. However, note that, in the wildcard placement mode, each AZ in the region must be over-provisioned appropriately to handle the tablets that it may receive from the other AZ's failover.
```

Comment:
, and `us-west-1` (AZs `us-west-1a`, `us-west-1b`,) and we wish to create an RF3 tablespace

Reply:
An example tablespace like below -> The tablespace below

Reply:
specifing -> specifying